### PR TITLE
Implement user_online API

### PR DESCRIPTION
**Pull request by phy25**
Now when the login is not done by this app, the balance can be fetched too.

Note that this hasn't be tested throughly. (It seems that it cannot be merged directly, so sorry for that)

### CODE REVIEW

**Comment by phy25** [comment]
Sorry but I see a problem: when using user_online API, I cannot see balance consumption. Only `user_in_bytes` and `user_out_bytes` varies. Maybe the old API behaves the same way.

So why are we refreshing the balance since it doesn't change after login 😂

**Comment by phy25** [comment]
Done.
- Now it ignores used bytes under 1M, so it's quite dirty.
- Don't know what will happen when balance is not enough.

File: app/src/main/java/org/bitnp/netcheckin2/util/NotifTools.java [hunk 1]
```diff
@@ -31,6 +35,17 @@ public static NotifTools getInstance(Context context){
         return instance;
     }
 
+    public static Bitmap getlogoBM(Context context){
```
Comment (LangleyChang) [inline app/src/main/java/org/bitnp/netcheckin2/util/NotifTools.java]: Explain the name.

Comment (phy25) [inline app/src/main/java/org/bitnp/netcheckin2/util/NotifTools.java]: Sorry, sometimes I am too lazy... Will fix this.

getLogoBM > getLogoBitmap

File: app/src/main/java/org/bitnp/netcheckin2/util/NotifTools.java [hunk 2]
```diff
@@ -31,6 +35,17 @@ public static NotifTools getInstance(Context context){
...
+        if(logoBM == null){
+            logoBM = BitmapFactory.decodeResource(context.getResources(), R.mipmap.logo);
```
Comment (LangleyChang) [inline app/src/main/java/org/bitnp/netcheckin2/util/NotifTools.java]: Decode bitmap really takes time.
Is this necessary or you can use `Context.getResources()` ?

**Comment by phy25** [comment]
I have been using this debug release for some time, and it seems not so
stable. (Strange things are happening.) I will find time later to look into
it.

2016-02-17 12:04 GMT+08:00 Zhehua Chang <notifications@github.com>:

**Comment by LangleyChang** [comment]
No hurry.
I'm going to refactor it thoroughly, and turn it into the 3rd version.
The current logic is too confusing.Because I didn't realize the complexity at the beginning, and it gets worse as extended and fixed.

As I used EventBus in onChart, I find it a great way to achieve this app, making the code neat and clear, for future modification.
I'll reorganize the logic and do some paper work for it firstly.Then, another repo I guess.
Advice and support are welcomed.Thanks.

**Pull request merged**